<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogPlot</class>
 <widget class="QMainWindow" name="LogPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1103</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogPlot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="openButton">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="fileName"/>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QCustomPlot" name="customPlot" native="true"/>
    </item>
    <item row="0" column="1" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>25</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="text">
         <string>Ctrl Log Plot</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>18</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Estimation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>13</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Desired</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="est_px">
            <property name="text">
             <string>p(x)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="est_py">
            <property name="text">
             <string>p(y)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="est_pz">
            <property name="text">
             <string>p(z)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="est_vx">
            <property name="text">
             <string>v(x)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="est_vy">
            <property name="text">
             <string>v(y)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="est_vz">
            <property name="text">
             <string>v(z)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="est_ax">
            <property name="text">
             <string>a(x)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="est_ay">
            <property name="text">
             <string>a(y)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="est_az">
            <property name="text">
             <string>a(z)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="est_yaw">
            <property name="text">
             <string>yaw</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="des_px">
            <property name="text">
             <string>p(x)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="des_py">
            <property name="text">
             <string>p(y)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="des_pz">
            <property name="text">
             <string>p(z)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="des_vx">
            <property name="text">
             <string>v(x)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="des_vy">
            <property name="text">
             <string>v(y)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="des_vz">
            <property name="text">
             <string>v(z)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="des_ax">
            <property name="text">
             <string>a(x)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="des_ay">
            <property name="text">
             <string>a(y)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="des_az">
            <property name="text">
             <string>a(z)</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>88</width>
              <height>17</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="plotButton">
          <property name="text">
           <string>Resize</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_5">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1103</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>LogPlot</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>960</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>963</x>
     <y>554</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
